docs/CLI.md

```diff
@@ -43,3 +43,9 @@ Options
   --yes, -y      confirm non-interactively
   --unmount, -u  unmount on success
 ```
+
+Debug mode
+----------
+
+You can set the `DEBUG` environment variable to make the Etcher CLI print error
```

Comment:
I just noticed that this "corresponds" nicely with the `ETCHER_CLI_ROBOT` environment variable used by `lib/shared/robot/index.js` - cool :sunglasses: